pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -229,6 +229,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
   checkFlags =
     [
       # Flaky: unreliably fails on certain hosts (including Hydra)
+      "--skip=zed::open_listener::tests::test_open_workspace_with_directory"
```

Comment:
Please put it behind `lib.optionals stdenv.hostPlatform.isDarwin` for now, until we get confirmation it also fails on Linux (which does not appear to be the case)